flexmeasures/api/v3_0/sensors.py

```diff
@@ -227,6 +232,26 @@ def get_data(self, sensor_data_description: dict):
         location="json",
     )
     @permission_required_for_context("create-children", ctx_arg_name="sensor")
+    @permission_required_for_context(
+        "read",
+        ctx_arg_name="flex_model",
+        ctx_loader=flex_model_loader,
```

Comment:
I'd add a comment: "This extracts sensors from the flex model parameter - user needs read access to them"